tests/test_error_cases.py

```diff
@@ -14,6 +16,16 @@
 
 CATALOG_PATH = data_dir / "sample_input_catalog.fits"
 
+COSMOS_CATALOG_PATHS = [
+    str(data_dir / "cosmos/real_galaxy_catalog_23.5_example.fits"),
+    str(data_dir / "cosmos/real_galaxy_catalog_23.5_example_fits.fits"),
+]
+
+COSMOS_EXT_CATALOG_PATHS = [
+    str(data_dir / "cosmos/real_galaxy_catalog_26_extension_example.fits"),
+    str(data_dir / "cosmos/real_galaxy_catalog_26_extension_example_fits.fits"),
+]
+
```

Comment:
These should be defined in the conftest.py to be used by any test.

Also make sure you separate the `"cosmos/xxx.fits"` into `"cosmos" / "xxx.fits"` so that pathlib can resolve Windows filepath, otherwise there is no point in using pathlib.